### PR TITLE
Implement switch account modal - Closes #4300

### LINKED ISSUE #4300: Implement switch account flow
### Description
Allow the user to switch between the stored accounts, user  working on application without decryption the phosphatase so public data will refetch they can’t sign transaction or any other functionality need the recovery phrase without decrypting the recovery phrase

Given users has added several accounts to their Lisk Wallet, they may switch between accounts as follows:
1. User requests the list of saved accounts.
2. local store provides the requested list.
3. User requests to switch between accounts.
4. User provides the selected account to Lisk Wallet.
5. Lisk Wallet switches to the selected account and refetch wallet data.

### Acceptance Criteria

- Allow users to switch between the account according to [design](https://www.figma.com/file/GbrsJiZlsh2OzVKqYDD4zI/Account-management-improvements?node-id=471%3A21289)
- Refetch data from current account
- Add e2e test to make sure data is fetch from the current account like balance wallet address transaction
- store current user on the redux

### Additional Information

-  User will allow decrypt the recovery phrase on transaction process we will ask for password to decrypt and use the recovery phrase to sign the transaction/messages process

### PR DESCRIPTION
**Pull request by isalga**
### What was the problem?

This PR resolves #4300

### How was it solved?

- Implement design
- Link to add account
- Close modal and dispatch login

### How was it tested?

Add unit test

### CODE REVIEW

File: src/modules/account/components/SwitchAccount/SwitchAccount.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```
Comment (soroushm) [inline src/modules/account/components/SwitchAccount/SwitchAccount.js:1]: I believe this component is duplicated 
you can reuse the ManageAccounts

**Review by soroushm — CHANGES_REQUESTED** [review]

File: src/modules/account/components/SwitchAccount/SwitchAccount.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import { useTranslation } from 'react-i18next';
+import ManageAccounts from '../ManageAccounts';
+
+const SwitchAccount = () => {
+  const { t } = useTranslation();
+
+  return (
+    <ManageAccounts title={t('Switch account')} />
```
Comment (soroushm) [inline src/modules/account/components/SwitchAccount/SwitchAccount.js]:
```suggestion
    <ManageAccounts title={t('Switch account')} isRemoveAvailable={false} />
```

**Comment by soroushm** [comment]
you need to reuse account manage component to a page  and make it center

**Comment by isalga** [comment]
How is it in design? it is a dialog. Do it you. You decide everything all the time and change team decitions constantly.

File: setup/jest.config.js [hunk 3]
```diff
@@ -52,7 +52,7 @@ module.exports = {
     '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
       '<rootDir>/tests/__mocks__/fileMock.js',
   },
-  collectCoverage: true,
+  collectCoverage: false,
```
Comment (ikem-legend) [inline setup/jest.config.js]: @isalga @eniolam1000752 Please revert coverage

**Review by ikem-legend — DISMISSED** [review]

**Review by soroushm — CHANGES_REQUESTED** [review]

File: src/modules/account/components/SwitchAccount/SwitchAccount.js (hunk 2 above)
Comment (isalga) [reply src/modules/account/components/SwitchAccount/SwitchAccount.js]: Changed, but I think both solutions are equally valid.

**Review by soroushm — APPROVED** [review]

File: src/modules/account/components/SwitchAccount/SwitchAccount.js (hunk 2 above)
Comment (soroushm) [reply src/modules/account/components/SwitchAccount/SwitchAccount.js]: I didn't notice changes related about this, could we have short call and explain to me?

**Review by ikem-legend — APPROVED** [review]

**Pull request merged**